tasks/build.js

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.registerTask('build',
+    'Build client',
+    ['lint', 'requirejs']);
```

Comment:
I don't see a requirejs task anywhere. If not needed, might as well remove this file.